__tests__/Str-test.js

```diff
@@ -129,10 +120,80 @@ describe('Str.fromCurrencyToNumber', () => {
 });
       
 describe('Str.isValidEmail', () => {
-    it('Correctly detects a valid email', () => {
+    it('Correctly identifies valid emails', () => {
         expect(Str.isValidEmail('abc@gmail.com')).toBeTruthy();
-        expect(Str.isValidEmail('test@gmail')).toBeFalsy();
-        expect(Str.isValidEmail('@gmail.com')).toBeFalsy();
-        expect(Str.isValidEmail('usernamelongerthan64charactersshouldnotworkaccordingtorfc822whichisusedbyphp@gmail.com')).toBeFalsy();
```

Comment:
All of these should still be invalid, right? Why not retain these tests?

Reply:
Though looks like at least the last one is already covered by the test on 184

Reply:
@NikkiWines @aimane-chnaif 
The cases have been restored.